### PR TITLE
Implement emulateWinChromiumPlatform Flag for OS Emulation

### PR DESCRIPTION
**Pull request by 3wweiweiwu**
**Challenge**

Some organizations use a Multi-Factor Authentication (MFA) application that supports only Windows and Mac systems. Users with unsupported operating systems, like Linux, are denied access due to system checks.

**Problem Analysis**

Web-based MFA applications often use properties such as `navigator.userAgent`, `navigator.userAgentData.platform`, `navigator.platform`, and `navigator.userAgentData.getHighEntropyValues` to detect the user's operating system. Although there is an option to modify the user agent, other properties also need emulation to fully mimic a Windows environment.

**Technical Details**

The implementation introduce a new PlatformEmulator class. It will be executed as part of preload script for chromium. This class will override specified property to emulate windows platform. 

**Solution**

Add the `emulateWinChromiumPlatform` flag to the CLI and configuration file, allowing users to opt-in to Windows platform emulation in Chromium browsers. This flag is turned off by default to avoid interference with existing workflows.

**Documentation**

Updated the README and the configuration documentation to include instructions on how to use the new `emulateWinChromiumPlatform` flag.

**Potential Impacts**

This change should not impact existing users as the default behavior remains unchanged. The feature is opt-in to ensure that only users needing this emulation enable it.

### CODE REVIEW

**Comment by IsmaelMartinez** [comment]
HI @3wweiweiwu , thanks a lot for contributing! I am hoping to release 1st the other PR (https://github.com/IsmaelMartinez/teams-for-linux/pull/1393). Then, if you could update the package.json file and the appdata (as described in https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/CONTRIBUTING.md#release-process) I can then merge it. But I will wait until the other PR is merged (so this would be released as 1.10.2). Thanks again!

**Comment by IsmaelMartinez** [comment]
1.10.1 is now out. If you can rebase and make the changes mentioned in my previous comment, then I can release this. (your version should be 1.10.2). Thanks!

**Comment by IsmaelMartinez** [comment]
looks good to me.

Can you update the package.json file and the appdata (as described in https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/CONTRIBUTING.md#release-process). I can't make the changes on your branch.

Then we can release it. Thanks again for your contribution!

File: com.github.IsmaelMartinez.teams_for_linux.appdata.xml [hunk 1]
```diff
@@ -14,6 +14,13 @@
 	<url type="bugtracker">https://github.com/IsmaelMartinez/teams-for-linux/issues</url>
 	<launchable type="desktop-id">com.github.IsmaelMartinez.teams_for_linux.desktop</launchable>
 	<releases>
+		<release version="1.10.2" date="2024-09-09">
```
Comment (IsmaelMartinez) [inline com.github.IsmaelMartinez.teams_for_linux.appdata.xml]:
```suggestion
		<release version="1.10.2" date="2024-09-10">
```

**Comment by IsmaelMartinez** [comment]
released! https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.10.2

**Comment by FranzXaver** [comment]
Thanks for this feature, this is exactly what I needed as my company also blocks Linux from MFA.
I had to also set the chromeUserAgent for it to work though, is this as exepected?

**Pull request merged**